setup_python_locally.sh

```diff
@@ -4,4 +4,5 @@ conda create -n tree-map
 conda activate tree-map
 conda config --env --add channels conda-forge
 conda config --env --set channel_priority strict
-conda install python=3 geopandas==0.4.0
\ No newline at end of file
+conda install python=3
+conda install --file requirements.txt
```

Comment:
@emillipede, in reference to your earlier question regarding setup scripts being in multiple places. This is one such location (the setup_python_locally.sh script). The other one lives in the python-env.sh, which references the environment.yml script:
https://github.com/Public-Tree-Map/public-tree-map-data-pipeline/blob/master/environment.yml

Reply:
It's not directly an argument for containers. It's more like having one setup script that runs locally and circleci.